src/app/ranking/page.tsx

```diff
@@ -0,0 +1,61 @@
+import { GiReturnArrow, GiCrown } from "react-icons/gi";
+
+export default function Home() {
+  const male = true;
+  const tabelaDados = [
+    { lugar: 1, players: "Player 1", score: 100 },
+    { lugar: 2, players: "Player 2", score: 90 },
+    { lugar: 3, players: "Player 3", score: 80 },
+    { lugar: 4, players: "Player 3", score: 80 },
+    { lugar: 5, players: "Player 3", score: 80 },
+  ];
+
+  return (
+    <div className="wrapper flex flex-col justify-between h-screen">
+      <header className="fl
+      ex justify-center pt-4 pl-4 pr-4">
```

Comment:
Ajustar esta quebra de linha (está quebrando o layout)
```suggestion
      <header className="flex justify-center pt-4 pl-4 pr-4">
```